accessibility-in-action/platform
# fix: create and enforce viewAny policies (resolves #789)

This PR adds restrictions so that users who have not verified their email, or who are (regulated) organization users with no association to a (regulated) organization, cannot view:

 - individuals
 - organizations
 - regulated organizations
 - projects
 - engagements

### Testing

As an:

- individual user with an unverified email
- organization user with an unverified email
- organization user who has not created/joined a organization
- regulated organization user with an unverified email
- regulated organization user who has not created/joined a regulated organization

1. Confirm that 'Projects' and 'People and Organizations' do not appear in the primary navigation;
2. Confirm that you cannot navigate to:
    - `/en/projects`
    - `/en/projects/all`
    - `/en/people-and-organizations`
    - '/en/individuals`
    - `/en/organizations`
    - `/en/regulated-organizations`
